app/main.py

```diff
@@ -7,13 +7,25 @@
     "The Lord of the Rings: The Return of the King",
     "Pulp Fiction",
     "The Good, the Bad and the Ugly",
-    "The Matrix",
+    "The Matrix"
 ]
 profile_info = ("michel", "michel@gmail.com", "12345678")
 marks = {
     "John": 4,
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
```

Comment:
The `collection_of_coins` is a set, which is an immutable type in Python. It should be categorized under 'immutable' instead of 'mutable'.